spython/tests/testdata/docker2singularity/entrypoint-cmd.def

```diff
@@ -0,0 +1,6 @@
+Bootstrap: docker
+From: busybox:latest
+%runscript
+exec python /code/script.py "$@"
```

Comment:
This is an inexact translation of the Dockerfile. See https://github.com/singularityhub/singularity-cli/issues/111#issuecomment-496805952

Reply:
Yes but this is what works and is expected for Singularity. You don't see users writing /bin/bash -c "..." as this is handled internally in Singularity.

Reply:
Completely missing the point. `docker run entrypoint-cmd --foo` does a `python --foo` not a `python /code/script.py --foo`

Reply:
It will need to be fixed then.